README.md

```diff
@@ -25,11 +25,11 @@ specified as ``fof(name, axiom, F)`` and ``fof(name, conjecture, F)`` respective
 ### Intuitionistic problems
 The collection of intuitionistic problems is described in [4] and it can be found in the directory  ``ILL``. The problems  were obtained from three main sources:
 
-1. Kleene's problems for intuitionistic logic from "Introduction to Metamathematics"
+1. Kleene's theorems for classic and intuitionistic logic from "Introduction to Metamathematics" (New York: van Nostrand, 1952)
 2. Intuitionistic Logic Theorem Provers library [ILTP](http://iltp.de/)
 3. Petri-nets from the [Model Checking Contest](https://pnrepository.lip6.fr/mcc/)
 
-The collections ``ILLTP-\*`` and ``KLE-\*`` are obtained via the translations call-by-name, call-by-value [1] and Liang and Miller's 0/1 [2],
+The collections ``ILLTP-\*`` and ``KLE-\*`` are obtained via the Girard translations in [1] and Liang and Miller's 0/1 [2],
```

Comment:
I would keep call-by-name and call-by-value (even if only in parentheses) since there are people who may identify more immediately the transformations by these names.